src/main/java/icoder/resources/TryResource.java

```diff
@@ -0,0 +1,142 @@
+package icoder.resources;
+
+import icoder.config.Config;
+import icoder.config.IConfig;
+import icoder.models.Service;
+
+import javax.inject.Inject;
+import javax.mail.Folder;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import java.util.ArrayList;
+import java.util.List;
+
+
+/**
+ * StatusResource controller
+ *
+ * @since 1.0
+ */
+@Path("/test")
+@Produces(MediaType.APPLICATION_JSON)
+public class TryResource {
+
+    @Inject
+    @IConfig
+    Config config;
+
+
+
+
+    @GET
+    public Service status() {
+
+        boolean a = this.checkIsNameDuplicateInParent(null);
+        System.out.println("TEST");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS106)